app/file.go

```diff
@@ -1356,6 +1356,12 @@ func (a *App) ExtractContentFromFileInfo(fileInfo *model.FileInfo) error {
 		if storeErr := a.Srv().Store.FileInfo().SetContent(fileInfo.Id, text); storeErr != nil {
 			return errors.Wrap(storeErr, "failed to save the extracted file content")
 		}
+		reloadFileInfo, storeErr := a.Srv().Store.FileInfo().Get(fileInfo.Id)
+		if storeErr != nil {
+			mlog.Warn("Failed to invalidate the fileInfo cache.", mlog.Err(storeErr), mlog.String("file_info_id", fileInfo.Id))
```

Comment:
I would suggest to return the error rather than just log as a warning. Cache invalidation can lead to subtle errors. It might be good to make this more visible.

Reply:
The point is, returning an error there give you the sensation that the error has happened, but that's not totally true, because the `SetContent` what is the main part, is already executed. This shouldn't fail ever, and in any case, is an improvement over the before lack of cache invalidation before.